library/src/main/java/com/novoda/downloadmanager/RoomDownloadsPersistence.java

```diff
@@ -56,7 +57,8 @@ public void persistBatch(DownloadsBatchPersisted batchPersisted) {
                     DownloadBatchIdCreator.createSanitizedFrom(roomBatch.id),
                     DownloadBatchStatus.Status.from(roomBatch.status),
                     roomBatch.downloadedDateTimeInMillis,
-                    roomBatch.notificationSeen
+                    roomBatch.notificationSeen,
+                    roomBatch.storageRoot
```

Comment:
This line right here! I forgot to add this one line and it seemed that none of the SQL statements were working for me. After a break and thinking I realised that the SQL statements were workings, it was just being overwritten when the app ran with `empty` because this was missing 😿